CMS-Enterprise/easi-app
# [EASI-4567] Team member notification emails

# EASI-4567

## Description

- add two new email templates for when users are added to teams on systems

## How to test this change

- add a new user to this team
- confirm email goes out to expected recipients based on ticket reqs

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
